ai_eval/templates/shortanswer.html

```diff
@@ -19,6 +19,9 @@
             </div>
         </div>
         <div class="submit-row">
+            {% if self.allow_reset %}
+                <span id="reset-button">Reset</span>
```

Comment:
Nit: I think we should name this `Reset Chat` or `Clear chat` instead of just `Reset` so the operation is clearer.